sos/report/plugins/collectd.py

```diff
@@ -28,7 +28,7 @@ def setup(self):
             '/etc/collectd.conf',
             '/etc/collectd.d/*.conf',
             '/var/log/containers/collectd/collectd.log',
-            '/var/lib/config-data/collectd',
```

Comment:
Can't `sos` run on some older version of `collectd` where the data are yet stored in this location?

Reply:
/var/lib/config-data/collectd should be a typo, it should have been /var/lib/config-data/puppet-generated/collectd in the first place. 
container based setups are the only setups used in OSP since OSP 10 was discontinued in December 2021